src/Sulu/Bundle/SecurityBundle/Entity/Role.php

```diff
@@ -127,6 +127,18 @@ public function getName()
         return $this->name;
     }
 
+    public function getKey()
+    {
+        return $this->key;
+    }
+
+    public function setKey($key)
+    {
+        $this->key = $key;
```

Comment:
Can roles with spaces be configured in the firewall e.g. `ROLE_TEST USER`?

Reply:
i think so, according to the docs a role only *must start with `ROLE_`*, otherwise its an arbitrary string:
https://symfony.com/doc/current/security.html#roles